adapters/http-vertx/src/test/java/org/eclipse/hono/adapter/http/impl/VertxBasedHttpProtocolAdapterTest.java

```diff
@@ -129,6 +130,8 @@
     @SuppressWarnings("unchecked")
     @BeforeClass
     public static void deployAdapter(final TestContext ctx) {
+
+        System.setProperty("vertx.logger-delegate-factory-class-name", SLF4JLogDelegateFactory.class.getName());
```

Comment:
If this is not required by the test, then I think it would make sense to put this into the maven build instead.

Reply:
Oh, that was a test only. I will remove that altogether ...